circle.yml

```diff
@@ -3,17 +3,19 @@ dependencies:
     - ~/.pyenv/versions/3.4.3
     - ~/coala-atom/node_modules
   pre:
-    # Install atom dependencies
```

Comment:
No need to remove this imo. If `libnotify4` is not related to atom-dependencies, just add another comment saying "Install non-atom dependencies" or "Install general dependencies"